src/txn/txn.c

```diff
@@ -1664,6 +1675,16 @@ __wt_verbose_dump_txn_one(WT_SESSION_IMPL *session, WT_TXN *txn)
       __wt_timestamp_to_string(txn->read_timestamp, ts_string[4]), txn->ckpt_lsn.l.file,
       txn->ckpt_lsn.l.offset, txn->full_ckpt ? "true" : "false",
       txn->rollback_reason == NULL ? "" : txn->rollback_reason, txn->flags, iso_tag));
+
+    /*
+     * Dump the information to standard error if error code is passed and return.
+     */
+    if (0 != error_code) {
+        WT_RET_MSG(session, error_code, "%s, %s", buf, error_string);
```

Comment:
What if the user doesn't pass an error string with the error code. How about:

`WT_RET_MSG(session, error_code, "%s %s", buf, error_string != NULL ? error_string : "" );